README.md

```diff
@@ -840,8 +840,8 @@ or email of the user who performed the maintenance task.
 
 ```ruby
 # config/initializers/maintenance_tasks.rb
-MaintenanceTasks.metadata = -> do
- { user_email: current_user.email }
+MaintenanceTasks.metadata = ->(context) do
+ { user_email: context.current_user.email }
```

Comment:
ahh got it! i will test it out and push some changes for review!

Reply:
lmk what you think of https://github.com/Shopify/maintenance_tasks/pull/873/commits/26f4a774c603bcdd5a1ed212a9af82ffe2c9fbe7 (passes a default empty block rather than `if/else`)! I tested locally with both the `MaintenanceTasks.metadata` set and without.